### DISCUSSION TITLE
Wie mit CO2-Äquivalenten umgehen?

### DISCUSSION THREAD

**Discussion by silberzwiebel** [codeformuenster/klimawatch#124] [General]
Aus einer Diskussion im Chat:

CO2-Äquivalente sollten am besten mit dargestellt werden. In der momentanen Berechnung, die auf [diesem Blogartikel](https://scilogs.spektrum.de/klimalounge/wie-viel-co2-kann-deutschland-noch-ausstossen/) basiert, werden CO2-Äquivalente _nicht_ berücksichtigt (s. [Kommentar](https://scilogs.spektrum.de/klimalounge/wie-viel-co2-kann-deutschland-noch-ausstossen/#comment-83274)).

- Gibt es überhaupt ein verlässliches CO2-Budget, welches die Äquivalente mit einberechnet? Die Tabelle im Blogartikel hat plusminus in der entsprechenden Spalte stehen.
- Welche Kommune gibt in ihrer Klimabilanz die Äquivalente mit an? Wie viel Arbeit ist es, das herauszufinden?
- Wie soll es visualisierungstechnisch umgesetzt werden?
  - Vorschlag: Entweder Äquivalente sind mit drin oder nicht. Je nach dem gibt es einfach nur eine andere Y-Achsen-Beschriftung.

**Comment by ccauet** [comment]
> Welche Kommune gibt in ihrer Klimabilanz die Äquivalente mit an?

Bei Landau weiß ich es, da ist CO2e explizit ausgewiesen.

> Wie soll es visualisierungstechnisch umgesetzt werden?

Ich denke wir müssen die Frage klären was es überhaupt für eine Auswirkung hat. Naiv stelle ich mich mal auf den Standpunkt: Wenn das Budget für reines CO2 ist, die Bilanz einer Stadt CO2e ausweist und bei der Umrechung nichts falsch gemacht wurde, dann dürfte das in erster Ordnung kein Problem sein, dass gegebene Budget weiter zu nutzen.

Wenn das so ist, dann reicht es vermutlich auch nur die Achsenbeschriftung zu ändern.

Wenn ich mir dann allerdings die Werte in der Tabelle angucke bei Non-CO2 scenario variation und Non-CO2 forcing and response uncertainty, dann vermute ich, dass das doch etwas zu vereinfacht ist. Die Unsicherheiten sind wirklich noch enorm groß.
